test/vtcomposite-internationalize.test.js

```diff
@@ -1,54 +1,56 @@
-const test = require('tape');
+'use strict';
+
 const internationalize = require('../lib/index.js').internationalize;
-const fs = require('fs');
-const path = require('path');
-const zlib = require('zlib');
+const { vtinfo, getFeatureById } = require('./test-utils.js');
+
 const mvtFixtures = require('@mapbox/mvt-fixtures');
-const vtinfo = require('./test-utils.js').vtinfo;
-const vt1infoValid = require('./test-utils.js').vt1infoValid;
-const tilebelt = require('@mapbox/tilebelt');
 
+const test = require('tape');
+const zlib = require('zlib');
 
-test('[internationalize] success: buffer size stays the same when no changes needed', function(assert) {
+test('[internationalize] success: buffer size stays the same when no changes needed', (assert) => {
   const singlePointBuffer = mvtFixtures.get('002').buffer;
 
-  internationalize(singlePointBuffer, 'piglatin', {}, (err, vtBuffer) => {
+  internationalize(singlePointBuffer, 'piglatin', null, (err, vtBuffer) => {
     assert.notOk(err);
+
+    const tile = vtinfo(vtBuffer);
+    assert.equal(tile.layers.hello.feature(0).id, undefined, 'ID should not be added to feature that does not have one');
```

Comment:
🧷 I ran this through a linter so there's lots of changes. I will try to point out where I expect a change of behavior

this line is a change, just verifying ID is not added